cdk/src/functions/living-docs-dashboard-generator.ts

```diff
@@ -1,38 +1,55 @@
-import { CloudWatchClient, PutDashboardCommand } from '@aws-sdk/client-cloudwatch';
+import { CloudWatchClient, DeleteDashboardsCommand } from '@aws-sdk/client-cloudwatch';
 import { SSMClient, GetParameterCommand } from '@aws-sdk/client-ssm';
 import endent from 'endent';
 
 const ssmClient = new SSMClient();
 const cloudwatchClient = new CloudWatchClient({});
 
-export async function handler(_event: any, _context: any) {
+const DOCS = `
```

Comment:
this is shown while the lambda is fetching the contents? I could not see it inside

Reply:
This is shown in the UI when you want to add a new custom widget.